pkg/parquetquery/intern/intern.go

```diff
@@ -7,72 +7,22 @@
 package intern
 
 import (
-	"unsafe"
+	"unique"
 
 	pq "github.com/parquet-go/parquet-go"
 )
 
-type Interner struct {
-	m map[string][]byte // TODO(mapno): Use swiss.Map (https://github.com/cockroachdb/swiss)
-}
+type Interner struct{}
 
 func New() *Interner {
-	return NewWithSize(0)
-}
-
-func NewWithSize(size int) *Interner {
-	return &Interner{m: make(map[string][]byte, size)}
+	return &Interner{}
 }
 
 func (i *Interner) UnsafeClone(v *pq.Value) pq.Value {
```

Comment:
Now that this is not "unsafe" we could change the method name to just Clone.  We can also remove this comment:

```
// It is not safe for concurrent use.
```

According to the documentation it's safe for concurrent use

Reply:
@mapno mentioned we might try to remove the package entirely.  My first attempt raised the allocations, but I think I see now the reason for that, so I'll try again shortly.

Reply:
I've pushed a commit to drop the `intern` package entirely.